aspnetcore/tutorials/web-api-help-pages-using-swagger/sample/TodoApi/TodoApi.csproj

```diff
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <DocumentationFile>bin\Debug\netcoreapp2.0\TodoApi.xml</DocumentationFile>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Folder Include="wwwroot\" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.All" Version="2.0.0" />
+    <PackageReference Include="Swashbuckle.AspNetCore" Version="1.0.0" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <DotNetCliToolReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Tools" Version="2.0.0" />
```

Comment:
I have two passing questions on this:
* Does the sample require code gen tooling? (I guess so, or else u wouldn't have it. lol)
* What happens if the sample is compiled for a non-Windows platform? ... Does this no-op or does it :boom:? ... just wondering.

Reply:
The code gen tooling was never used. It was there in the project template I used. I still need to test this on a non-Windows platform.

Reply:
Reminds me: My *ancient* Mac Mini can't run 2.0 b/c it's so freak'in old that it can't take the Sierra update! Yikes! My Mac Mini should be in the *Smithsonian*. :smile: ... I'll have to look into getting a new one.

Reply:
@guardrex If nothing else, that Mac Mini is a great conversation starter.